tests/integration/operatorhub/cmd_link_test.go

```diff
@@ -32,7 +33,7 @@ var _ = Describe("odo link command tests for OperatorHub", func() {
 		BeforeEach(func() {
 			// wait till odo can see that all operators installed by setup script in the namespace
 			odoArgs := []string{"catalog", "list", "services"}
-			operators := []string{"redis-operator", "service-binding-operator"}
```

Comment:
It should be better to adapt the value of `operators` depending on the value of `os.Getenv("KUBERNETES")`